iot-hub/Samples/device/PnpDeviceSamples/PnpConvention/PnpConvention.cs

```diff
@@ -170,25 +163,25 @@ public static string CreateComponentPropertyPatch(string componentName, string p
         /// </code>
         /// </remarks>
         /// <param name="componentName">The name of the component in which the property is defined. Can be null for property defined under the root interface.</param>
-        /// <param name="propertyKeyValuePairs">The property name and an unserialized value, as defined in the DTDL interface.</param>
+        /// <param name="propertyPairs">The property name and an unserialized value, as defined in the DTDL interface.</param>
         /// <returns>The property patch for read-only and read-write property updates.</returns>
-        public static string CreateComponentPropertyPatch(string componentName, IDictionary<string, object> propertyKeyValuePairs)
+        public static string CreateComponentPropertyPatch(string componentName, IDictionary<string, object> propertyPairs)
         {
             if (string.IsNullOrWhiteSpace(componentName))
             {
                 throw new ArgumentNullException(nameof(componentName));
             }
-            if (propertyKeyValuePairs == null)
+            if (propertyPairs == null)
             {
-                throw new ArgumentNullException(nameof(propertyKeyValuePairs));
+                throw new ArgumentNullException(nameof(propertyPairs));
             }
 
             var propertyPatch = new StringBuilder();
             propertyPatch.Append("{");
             propertyPatch.Append($"\"{componentName}\":");
             propertyPatch.Append("{");
             propertyPatch.Append($"\"{PropertyComponentIdentifierKey}\":\"{PropertyComponentIdentifierValue}\",");
-            foreach (var kvp in propertyKeyValuePairs)
+            foreach (var kvp in propertyPairs)
             {
                 propertyPatch.Append($"\"{kvp.Key}\": {JsonConvert.SerializeObject(kvp.Value)},");
```

Comment:
super nitty: extra space between the key and value (in the string interpolation).